Prac_07/KivyDemos-master/convert_miles_km.py

```diff
@@ -0,0 +1,35 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.core.window import Window
+from kivy.app import StringProperty
+
+MILES_TO_KM = 1.609
+
+
+class ConvertDistanceApp(App):
+    message = StringProperty()
+
+    def build(self):
+        Window.size = (500, 200)
+        self.title = "Converting Distances"
+        self.root = Builder.load_file('convert_miles_km')
+        return self.root
+
+    def convert_miles(self):
+        valid = self.check_valid()
```

Comment:
valid as a variable, sounds like it's a boolean. The variable contains a distance, so i would stick with that.